### DISCUSSION TITLE
BioTech

### DISCUSSION THREAD

**Discussion by brunoyillli** [Ludeon/RimWorld-PortugueseBrazilian#260] [General]
eu tentei instalar com o script de vocês, mas a parte de biotech continua toda em inglês, ainda está em progresso ?

**Comment by ElioNETO-trad** [comment]
Olá Bruno,

sim, a expansão foi lançada há poucos dias, apenas começamos a trabalhar na tradução. É um trabalho voluntário, então traduzimos de acordo com nossas disponibilidades. No entanto, a tradução segue ativa e será publicada!

Elio NETO
